pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
+ suffix = ".tar.gz";
+
+ meta = with stdenv.lib; {
+   description = "RedWax CA service modules.";
+   version = "0.2.1";
+
+   homepage = "https://redwax.eu";
+   license = licenses.asl20;
+   platforms = platforms.unix;
+   maintainers = with maintainers; [ dirkx ];
+ };
+
+ src = fetchurl {
+   url = "${baseurl}${name}-${meta.version}${suffix}";
+   sha256 = "1pxapjrzdsk2s25vhgvf56fkakdqcbn9hjncwmqh0asl1pa25iic";
+ };
+
+ buildInputs = [ gnused coreutils pkgconfig apacheHttpd openssl openldap ];
```

Comment:
As mentioned earlier `gnused` and `coreutils` aren't needed because they are already provided for you, just like `gcc`, which you didn't have to explicitly list.

Reply:
fixed everywhere.